sdks/java/io/elasticsearch/pom.xml

```diff
@@ -133,6 +123,12 @@
 
     <dependency>
       <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
```

Comment:
-1: as elasticsearch IO uses SLF4J Logger, it's not a test dependency, it's actually used in the IO.